packages/oscal-react-library/src/components/OSCALCatalog.tsx

```diff
@@ -15,6 +24,11 @@ export default function OSCALCatalog(props) {
     },
   };
 
+  const defaultGroup: ControlGroup = {
+    title: "*Ungrouped Controls*",
```

Comment:
Can we make "*Ungrouped Controls*" a constant somewhere? I feel like we may want to change this at some point.